docs/tutorials/trainers.ipynb

```diff
@@ -263,67 +265,24 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "73700fb5",
    "metadata": {
     "id": "73700fb5"
    },
    "source": [
-    "We load the log files and plot the training RMSE over batches, and the validation RMSE over epochs. We can see that our model is just starting to converge, and would probably benefit from additional training time and a lower initial learning rate."
+    "We launch TensorBoard to visualize RMSE over batches, and the validation RMSE over epochs as well as other information about our model. We can see that our model is just starting to converge, and would probably benefit from additional training time and a lower initial learning rate."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c38033e0",
-   "metadata": {
-    "id": "c38033e0"
-   },
-   "outputs": [],
-   "source": [
-    "if not fast_dev_run:\n",
-    "    train_steps = []\n",
-    "    train_acc = []\n",
-    "\n",
-    "    val_steps = []\n",
-    "    val_acc = []\n",
-    "\n",
-    "    with open(\n",
-    "        os.path.join(default_root_dir, \"tutorial_logs\", \"version_0\", \"metrics.csv\")\n",
-    "    ) as f:\n",
-    "        csv_reader = csv.DictReader(f)\n",
-    "        for i, row in enumerate(csv_reader):\n",
-    "            try:\n",
-    "                train_acc.append(float(row[\"train_OverallAccuracy\"]))\n",
-    "                train_steps.append(i)\n",
-    "            except ValueError:  # Ignore rows where train RMSE is empty\n",
-    "                pass\n",
-    "\n",
-    "            try:\n",
-    "                val_acc.append(float(row[\"val_OverallAccuracy\"]))\n",
-    "                val_steps.append(i)\n",
-    "            except ValueError:  # Ignore rows where val RMSE is empty\n",
-    "                pass"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "1ba23407",
-   "metadata": {
-    "id": "1ba23407"
-   },
+   "id": "3e95ee0a",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "if not fast_dev_run:\n",
-    "    plt.figure()\n",
-    "    plt.plot(train_steps, train_acc, label=\"Train Accuracy\")\n",
-    "    plt.plot(val_steps, val_acc, label=\"Val Accuracy\")\n",
-    "    plt.legend(fontsize=15)\n",
-    "    plt.xlabel(\"Batches\", fontsize=15)\n",
-    "    plt.ylabel(\"Accuracy\", fontsize=15)\n",
-    "    plt.show()\n",
-    "    plt.close()"
+    "%tensorboard --logdir /tmp/experiments/tutorial_logs"
```

Comment:
This only works because `tempfile.gettempdir()` returns `/tmp` on Linux. On macOS, it returns `/var/folders/jv/cgkfvslj6nq1l7cw0c8c_8gm0000gn/T` (random hash each time), so the log files won't be found.

Is there any way to use the `default_root_dir` Python variable here? If not, we may have to choose a local directory to ensure we know where it will be.

Reply:
I believe we can use `%tensorboard --logdir "$default_root_dir"`.